control-plane-operator/controllers/hostedcontrolplane/kcm/deployment.go

```diff
@@ -327,7 +327,9 @@ func kcmArgs(p *KubeControllerManagerParams) []string {
 		"--kube-api-qps=150",
 		"--leader-elect-resource-lock=configmapsleases",
 		"--leader-elect=true",
-		"--leader-elect-retry-period=3s",
+		"--leader-elect-retry-period=26s",
```

Comment:
Could you please add a small unit test for these everywhere you added them?

Reply:
It looks like this file doesn't include any unit test at the moment, how do you want me to test this bit?

Reply:
Yes, please do

Reply:
Can you explain how you want me to test it?

Reply:
The easiest might be to factor the arguments into a function, then write a test that iterates over the arguments and ensure that there is only one occurance of each of the args and that this one occurence has the correct value

Reply:
See https://github.com/openshift/hypershift/pull/1427 for an example regarding how this can be tested

Reply:
Testing fucntion added